WireUI/Sources/WireIndividualToTeamMigrationUI/Components/PageContainer.swift

```diff
@@ -53,12 +53,14 @@ struct PageContainer<Content: View>: View {
                     Spacer(minLength: 36)
```

Comment:
Unrelated to these changes here, I posted the same on the original [PR](https://github.com/wireapp/wire-ios/pull/2284/files/919fd77e96771b294353a286ddf43fcf458727d4#r1881957136):
Do we (= the designers) want the bottom button to scroll as well or let only the content above the button scroll?
Anyway, it's probably too much effort to change here, right?

Reply:
@caldrian Certainly too much effort to change here but I agree - I think the button shouldn't scroll. I looked into this a small amount and believe that this is achievable using the `safeAreaInset()` modifier: https://www.hackingwithswift.com/quick-start/swiftui/how-to-inset-the-safe-area-with-custom-content.